<ui version="4.0" >
 <class>newsDialog</class>
 <widget class="QDialog" name="newsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Arch Linux News</string>
  </property>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <item>
     <widget class="QTreeWidget" name="treeWidget" >
      <property name="indentation" >
       <number>0</number>
      </property>
      <column>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="shaman.qrc" >
         <normaloff>:/Icons/icons/mail-mark-unread-new.png</normaloff>:/Icons/icons/mail-mark-unread-new.png</iconset>
       </property>
      </column>
      <column>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="shaman.qrc" >
         <normaloff>:/Icons/icons/mail-message-new.png</normaloff>:/Icons/icons/mail-message-new.png</iconset>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>News-Title</string>
       </property>
       <property name="icon" >
        <iconset resource="shaman.qrc" >
         <normaloff>:/Icons/icons/draw-text.png</normaloff>:/Icons/icons/draw-text.png</iconset>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>Link</string>
       </property>
       <property name="icon" >
        <iconset resource="shaman.qrc" >
         <normaloff>:/Icons/icons/application-rss+xml.png</normaloff>:/Icons/icons/application-rss+xml.png</iconset>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>status</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label" >
      <property name="text" >
       <string/>
      </property>
      <property name="pixmap" >
       <pixmap/>
      </property>
      <property name="alignment" >
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QPushButton" name="asReadButton" >
        <property name="text" >
         <string>Mark as Read</string>
        </property>
        <property name="icon" >
         <iconset resource="shaman.qrc" >
          <normaloff>:/Icons/icons/mail-mark-unread-new.png</normaloff>:/Icons/icons/mail-mark-unread-new.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openButton" >
        <property name="text" >
         <string>Open in Browser</string>
        </property>
        <property name="icon" >
         <iconset resource="shaman.qrc" >
          <normaloff>:/Icons/icons/document-open.png</normaloff>:/Icons/icons/document-open.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="refreshButton" >
        <property name="text" >
         <string>Refresh</string>
        </property>
        <property name="icon" >
         <iconset resource="shaman.qrc" >
          <normaloff>:/Icons/icons/application-rss+xml.png</normaloff>:/Icons/icons/application-rss+xml.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="closeButton" >
        <property name="text" >
         <string comment="close the dialog" >Close</string>
        </property>
        <property name="icon" >
         <iconset resource="shaman.qrc" >
          <normaloff>:/Icons/icons/application-exit.png</normaloff>:/Icons/icons/application-exit.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="shaman.qrc" />
 </resources>
 <connections/>
</ui>
